### DISCUSSION TITLE
Getting cable terminations details via GraphQL

### DISCUSSION THREAD

**Discussion by derdeagle** [netbox-community/netbox#14583] [Help Wanted!]
Hi all,

Could anybody please tell me how I can retrieve cable terminations details via GraphQL?
A shortened example GraphQL query I am using is e.g.

```
{
  device_list {
    name
    interfaces {
      name
      cable {
        label
      }
    }
  }
}
```

I can get the label of a cable but how can I get the terminations, like device name and interface?
When using the autocomplete feature I get the `terminations` key but under that I don't get anything (or at least I don't see it) regarding the terminations details.

**Comment by ianbarrere** [comment]
@derdeagle did you ever manage to figure this out? I'm trying to do the same thing, and currently stuck.

**Reply by derdeagle** [reply]
Unfortunately not, I switched to the standard API calls to get the information I need. Even the example you posted below didn't work for me.

**Comment by ianbarrere** [comment]
I sort of pieced this together by using a web browser. I didn't realize, but it opens graphiql which is a nicer way to explore the schema, it has autocompletion and nice things like that. Anyway, this is what I settled on:

```
{
  device_list(name: "<device_name>") {
    id
    name
    interfaces {
      name
      ip_addresses {
        address
      }
      cable {
        terminations {
          _device {
            name
          }
        }
      }
    }
  }
}
```
